tests/integ/test_tfs.py

```diff
@@ -83,6 +83,30 @@ def tfs_predictor_with_model_and_entry_point_same_tar(instance_type,
         yield predictor
 
 
+@pytest.fixture(scope='module')
+def tfs_predictor_with_model_and_entry_point_different_naming(instance_type,
```

Comment:
that is long, but I have no other idea lol

Reply:
It is okay to be long. Some other ideas using test naming conventions: https://dzone.com/articles/7-popular-unit-test-naming

Let me know if you have any suggestions, I will keep the name otherwise.